### PR TITLE
properly escape quotes in email addresses

### PR DESCRIPTION
**Pull request by mwvolo**
https://openstax.sentry.io/issues/5414102669/?environment=production&project=5772471&query=is%3Aunresolved%20issue.priority%3A%5Bhigh%2C%20medium%5D&referrer=issue-stream&statsPeriod=24h&stream_index=1

### CODE REVIEW

**Comment by mwvolo** [comment]
after this is merged, let's get master renamed to main

**Comment by mwvolo** [comment]
nvm, updated the PR when I renamed, all good

**Review by Dantemss — APPROVED** [review]

**Comment by mwvolo** [comment]
I'm having some specs fail locally re: quote escaping. Looking into it - hold off on merging.

File: spec/active_force/active_query_spec.rb
Comment (Dantemss) [inline spec/active_force/active_query_spec.rb:150]: This and line 164 should still give the same result if you use only 2 backslashes instead of 3

Comment (mwvolo) [reply spec/active_force/active_query_spec.rb:150]: indeed it does - i'll push an update

Comment (mwvolo) [reply spec/active_force/active_query_spec.rb:150]: I just pushed to main - https://github.com/openstax/active_force/commit/3210375b8ea95b576c876de87f1e02fcdc0139d0

**Pull request merged**
